Tensile/SolutionStructs.py

```diff
@@ -1758,6 +1800,9 @@ def assignDerivedParameters(state):
     if state["GlobalReadVectorWidth"] == -1:
       state["GlobalReadVectorWidth"] = state["VectorWidth"]
 
+    # Default GlobalStoreVectorWidth
+    if state["StoreVectorWidth"] == -1:
+      state["StoreVectorWidth"] = 2  if state["ProblemType"]["DataType"].numBytes() == 2 else 4
```

Comment:
should the default be VectorWidth?
Common.py says this is #elements but this looks like bytes?
What about doubles?